sklearn/feature_selection/_univariate_selection.py

```diff
@@ -857,13 +850,12 @@ class GenericUnivariateSelect(_BaseFilter):
     SelectFwe : Select features based on family-wise error rate.
     """
 
-    _selection_modes = {'percentile': SelectPercentile,
-                        'k_best': SelectKBest,
+    _selection_modes = {'percentile': SelectPercentile,  # type: ignore
```

Comment:
these are unrelated, right? Sorry I'm not up to date with what we do with type annotation...

Reply:
Yeah, these look unrelated. mypy might start complaining because the PR affects many files that have not been updated recently. I also don't know what is the preferred way to deal with mypy.